### PR TITLE
Update sqlx to 0.7.2

### PR DESCRIPTION
**Pull request by v0idpwn**

### CODE REVIEW

**Comment by v0idpwn** [comment]
@pbzweihander I rebased your PR and did some chore since 0.7.2 was released. Do you think its good to go?

**Comment by pbzweihander** [comment]
I think it is good to go!

File: pgmq-rs/Cargo.toml [hunk 1]
```diff
@@ -12,13 +12,13 @@ readme = "README.md"
...
-pgmq_core = { package = "pgmq-core", version = "0.6.1" }
+pgmq_core = { package = "pgmq-core", path = "../core" }
 chrono = { version = "0.4.23", features = [ "serde" ] }
 serde = { version = "1.0.152" }
 serde_json = { version = "1.0.91", features = [ "raw_value" ] }
-sqlx = { version = "0.6.3", features = [ "offline", "runtime-tokio-native-tls" , "postgres", "chrono", "json" ] }
+sqlx = { version = "0.7.2", features = [ "runtime-tokio-native-tls" , "postgres", "chrono", "json" ] }
 thiserror = "1.0.38"
-tokio = { version = "1", features = ["macros"] }
+tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
```
[the remark this answers is not in the document]
Comment (v0idpwn) [reply pgmq-rs/Cargo.toml]: Without this feature in tokio, I get the following error: https://github.com/tembo-io/pgmq/actions/runs/6389164714/job/17340109202

Is there a better solution?

[the remark this answers is not in the document]
Comment (v0idpwn) [reply pgmq-rs/Cargo.toml]:
```
Couldn't compile the test.
---- src/lib.rs - PGMQueue::set_vt (line 873) stdout ----
error: The default runtime flavor is `multi_thread`, but the `rt-multi-thread` feature is disabled.
  --> src/lib.rs:884:1
   |
13 | #[tokio::main]
   | ^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `tokio::main` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error
```

**Review by vrmiguel — APPROVED** [review]

**Review by vrmiguel — APPROVED** [review]

**Pull request merged**